protocol/x/clob/keeper/process_operations.go

```diff
@@ -696,21 +679,6 @@ func (k Keeper) PersistMatchDeleveragingToState(
 				err,
 			)
 		}
-
-		if quoteQuantums, err := k.perpetualsKeeper.GetNetNotional(
-			ctx,
-			perpetualId,
-			new(big.Int).SetUint64(fill.FillAmount),
-		); err == nil {
-			labels := []gometrics.Label{
-				metrics.GetLabelForIntValue(metrics.PerpetualId, int(perpetualId)),
-			}
-			telemetry.IncrCounterWithLabels(
-				[]string{metrics.ProcessOperations, metrics.Deleveraging, metrics.QuoteQuantums},
-				metrics.GetMetricValueFromBigInt(quoteQuantums),
-				labels,
-			)
-		}
```

Comment:
also duplicated - there is a metric in process deleveraging that keeps track of amount deleveraged